openai-core/src/commonMain/kotlin/com.aallam.openai.api/assistant/AssistantRequest.kt

```diff
@@ -38,16 +37,46 @@ public data class AssistantRequest(
     @SerialName("tools") val tools: List<AssistantTool>? = null,
 
     /**
-     * A list of file IDs attached to this assistant. Optional. Defaults to an empty list.
-     * There can be a maximum of 20 files attached to the assistant.
+     * A set of resources that are used by the assistant's tools. The resources are specific to the type of tool.
+     * For example, the code_interpreter tool requires a list of file IDs, while the file_search tool requires a list of vector store IDs.
      */
-    @SerialName("file_ids") val fileIds: List<FileId>? = null,
+    @SerialName("tool_resources") val toolResources: ToolResources? = null,
```

Comment:
Hey man @aallam , really appreciate what you have been creating. I realized that after this change many things were not replaced properly: 
https://github.com/aallam/openai-kotlin/blob/629d751b5b48648234682b7145e592557f35d0f6/sample/jvm/src/main/kotlin/com/aallam/openai/sample/jvm/AssistantsRetrieval.kt#L27C9-L33

and that might be confusing for many devs. just for your information. 
thank you again, this library is just great